### PR TITLE
Fix: Removed upload csv buttons

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Remove import CSV buttons

Task: [ID]()

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

**Review by Fabricevladimir — COMMENTED** [review]
There's an issue with `MintTokenDescription` Component, can you do the link in html fix that James implemented?

**Comment by sepehr2github** [comment]
> There's an issue with the `MintTokenDescription` Component, can you do the link in html fix that James implemented?

Thanks for mentioning, But in the new crowdin json I couldn't find any link regarding this so I removed the components and added a simple translation as a replacement

File: packages/web-app/src/containers/actionBuilder/mintTokens/index.tsx [hunk 1]
```diff
@@ -343,30 +343,30 @@ export const MintTokenForm: React.FC<MintTokenFormProps> = ({
...
-              append({address: tuple[0], amount: tuple[1]});
-            }
-          }
-        }
-      };
-
-      reader.readAsBinaryString(myFile);
-    }
-  };
+  // const handleCSVUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/actionBuilder/mintTokens/index.tsx:346]: Funny thing is, I'm pretty sure this worked lol

**Review by Fabricevladimir — APPROVED** [review]
Thanks, Sepehr. Good to merge

**Pull request merged**